src/components/App/App.js

```diff
@@ -50,68 +49,60 @@ class App extends Component {
           return pokemon;
         }
       }
-    });
-
+    })
+    console.log(verifiedName, '<><><><>')
+// This condition not working
     if (verifiedName === undefined) {
-      return this.setState({ error: "Not  a valid Name or id , try again" });
+      setError("Not  a valid Name or id , try again");
```

Comment:
We definitely  need to figure the why here... we will do it! Let's rock this one as well